frame/babe/src/lib.rs

```diff
@@ -188,6 +189,9 @@ decl_storage! {
 		// variable to its underlying value.
 		pub Randomness get(fn randomness): schnorrkel::Randomness;
 
+		/// The Current epoch configuration.
+		EpochConfig: Option<BabeEpochConfiguration>;
```

Comment:
Is there a way to remove the `Option` here without making `BabeEpochConfiguration` `Default`?

Reply:
How should it be set from the `GenesisConfig`?

Reply:
No. If the storage item is not an Option, then it must implement a default state, which is what will be returned when you query that storage item when it hasn't been set.

To enable genesis config, you can add `config()`, which will directly allow you to set some value in the genesis config, you you can add `build()` which will allow you to do more complex kind of setup, for example allowing a user to config some variables which then get built into the final storage item.

You can peek at the syntax documentation here: https://crates.parity.io/frame_support/macro.decl_storage.html